app/controllers/todos_controller.rb

```diff
@@ -1,7 +1,21 @@
 class TodosController < ApplicationController
+    @@sort_Type = "none"
     before_action :authenticate
     def index
-        @todos = current_user.todos
+        if params[:sort]
+            @@sort_Type = params[:sort]
+        end
+        if @@sort_Type == "none"
```

Comment:
Can we use a more efficient way of finding sort modes than `if` and `elsif` statements?

Reply:
Ok so this will be changed and refactored entirely by just calling and passing directly from the database rather than trying to store data inside of @todos which will make things much smoother.